tss.el

```diff
@@ -533,7 +533,20 @@
                (tss--trace "Got response from server.")
                tss--server-response))))))
 
-(defun* tss--sync-server (&key waitsec path buff)
+(defun* tss--sync-server (&key waitsec buff path
+                               source linecount)
+  "Sync current source with tss service. Default to 15 waiti,
```

Comment:
Abbrebriate for "wait Interval" I believe... I follow the convention is the code. Refer to the code below there is waiti and maxwaiti, and they are used as counter in the while loop further down. So I think actually "waitsec" is the misleading name here (it's not used as secs ...).
(PS: just naming issue, not elisp specific)